parm/metplus_config/stats/mesoscale/precip/PcpCombine_obsCCPA.conf

```diff
@@ -1,22 +1,21 @@
 [config]
 #### Input
-INPUT_BASE = {ENV[DATA]}/data
+INPUT_BASE = {ENV[DATA]}/{ENV[VERIF_CASE]}
 FCST_PCP_COMBINE_RUN = False
 OBS_PCP_COMBINE_RUN = True
-OBS_PCP_COMBINE_INPUT_DIR = {INPUT_BASE}/ccpa
+OBS_PCP_COMBINE_INPUT_DIR = {INPUT_BASE}/data/ccpa
 OBS_PCP_COMBINE_INPUT_TEMPLATE = ccpa.accum0{ENV[ccpa_file_accum]}hr*
 OBS_PCP_COMBINE_INPUT_DATATYPE = GRIB
 OBS_IS_PROB = False
 #### Output
-OUTPUT_BASE = {ENV[DATA]}
-LOG_DIR = {OUTPUT_BASE}/logs
+FCST_PCP_COMBINE_OUTPUT_DIR = {ENV[job_num_work_dir]}
```

Comment:
Should be `OBS_PCP_COMBINE_OUTPUT_DIR`, then remove line 14 with `OBS_PCP_COMBINE_OUTPUT_DIR = {OUTPUT_BASE}`, change line 12 to `LOG_DIR = {OBS_PCP_COMBINE_OUTPUT_DIR}/logs`, and line 17 to `METPLUS_CONF = {OBS_PCP_COMBINE_OUTPUT_DIR}/confs/{ENV[JOB_GROUP]}_job{ENV[job_num]}_for{VALID_BEG}_runon{CLOCK_TIME}.conf`

Reply:
Thank you, good point!  Updated.